docs/tracing/sdks/java.md

```diff
@@ -0,0 +1,393 @@
+title: Java OpenTelemetry SDK for Sematext Tracing
+description: Complete guide to instrumenting Java applications with OpenTelemetry for Sematext Tracing
+
+## Java OpenTelemetry SDK
+
+This guide covers how to instrument Java applications with OpenTelemetry to send traces to Sematext Tracing.
+
+> **Note**: Kotlin applications can use the same Java instrumentation. All examples below work with Kotlin code.
+
+## Auto-Instrumentation (Recommended)
+
+Auto-instrumentation is the easiest way to get started. The OpenTelemetry Java agent automatically instruments popular frameworks and libraries without requiring code changes.
+
+### 1. Download the OpenTelemetry Java Agent
+
+```bash
+wget https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+```
+
+### 2. Run Your Application with the Agent
+
+```bash
+java -javaagent:opentelemetry-javaagent.jar \
+  -Dotel.service.name=your-service-name \
+  -Dotel.exporter.otlp.endpoint=http://localhost:4338 \
+  -Dotel.exporter.otlp.protocol=http/protobuf \
+  -jar your-application.jar
+```
+
+### Framework-Specific Considerations
+
+**Spring Boot**
+
+- No additional configuration needed
+- Auto-instrumentation works out of the box
+- Captures HTTP requests, database calls, and more
+
+**Tomcat**
+
+- Works with WAR deployments
+- Add the agent to your Tomcat startup script
+
+**Jetty**
+
+- Supported out of the box
+- Add the agent to your Jetty configuration
+
+### Environment Variables
+
+You can also configure the agent using environment variables:
+
+```bash
+export OTEL_SERVICE_NAME=your-service-name
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4338
+export OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf
+
+java -javaagent:opentelemetry-javaagent.jar -jar your-application.jar
+```
+
+## Manual Instrumentation (Advanced)
+
+For more control over instrumentation, you can manually instrument your application.
+
+### 1. Add Dependencies
+
+**Maven (pom.xml)**:
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-api</artifactId>
+    <version>1.32.0</version>
```

Comment:
So we will want to update the these docs periodically as OTel SDK versions change, eh? And in SC itself, too?

Reply:
Periodically, yes but not every after single update.